### ISSUE TITLE
error running call reference ban button

### ISSUE THREAD

**Issue by raynaldograndy** [kanersps/es_admin#10]
hello, this error always appear when i press ban button on es_admin interface

![Capture](https://user-images.githubusercontent.com/15141130/56915641-a2a29900-6ae9-11e9-93b8-d217de020018.PNG)

**Comment by ArnaudLier** [comment]
it's doing that for me too...

**Comment by Oreax** [comment]
Ever found a solution for this? Mine started doing it, but I have no idea what specifically I've changed in the meantime.

**Comment by kanersps** [comment]
Seems like a bit weird of an error, did you try banning someone who already left the server?

**Comment by Oreax** [comment]
Nah, I tested it on my dev server - same thing there with a person who's online. /ban works though, just not using **0** for permban, as I understand it, that's how it's supposed to go? But it take 0 seconds and it's basically a kick :)
